_overviews/compiler-options/optimizer.md

```diff
@@ -0,0 +1,223 @@
+---
+layout: singlepage-overview
+title: Optimizer
+---
+
+**[Lukas Rytz](https://github.com/lrytz) (November 7, 2018)**
+
+**[Andrew Marki](https://github.com/som-snytt) (February 29, 2022)**
```

Comment:
s/February 29, 2022/2022/

and I'll have you know that I SEE WHAT YOU DID THERE, THERE IS NO FEBRUARY 29 2022 AND WE FROWN ON THESE SHENANIGANS

Reply:
I literally can't get anything by you. I mean, a doc PR during the long tail of pandemic when there's no one available to do reviews, I thought I could sneak in one chuckle.

Reply:
THERE SHALL BE NO HUMOR IN SCALA